apps/challenges/aws_utils.py

```diff
@@ -1118,7 +1122,6 @@ def setup_eks_cluster(challenge):
     Arguments:
         instance {<class 'django.db.models.query.QuerySet'>} -- instance of the model calling the post hook
     """
-    from .models import ChallengeEvaluationCluster
     from .serializers import ChallengeEvaluationClusterSerializer
```

Comment:
@burnerlee please move serializer and utils imports to the start of the file and remove imports from the methods.